web/crux/src/app/registry/registry.service.ts

```diff
@@ -87,6 +87,13 @@ export default class RegistryService {
       },
     })
 
-    return this.mapper.detailsToGrpc(registry)
+    const images = await this.prisma.image.count({
```

Comment:
It could be resolved in the same query
`
    const registry = await this.prisma.registry.findUniqueOrThrow({
      include: {
        _count: {
          select: {
            images: true,
          },
        },
      },
      where: {
        id: req.id,
      },
    })
`